lib/pine.coffee

```diff
@@ -11,6 +11,10 @@ class ResinPine extends PinejsClientCore
 		params.json = true
 		params.gzip ?= true
 
+		# Support "data" attribute
+		if params.data? and not params.body?
```

Comment:
`data` is never passed to `_request` by pinejs-client, so this code will never be triggered in normal use (only if someone explicitly calls `_request` which is unsupported)